jobs/payment-jobs/tasks/cfs_create_account_task.py

```diff
@@ -124,15 +124,21 @@ def _create_cfs_account(cls, pending_account: CfsAccountModel, pay_account: Paym
 
         except Exception as e:  # NOQA # pylint: disable=broad-except
             # publish to mailer queue.
+            is_user_error = False
             if pay_account.payment_method == PaymentMethod.PAD.value:
                 mailer.publish_mailer_events('PadSetupFailed', pay_account)
```

Comment:
Don't we need move `mailer.publish_mailer_events('PadSetupFailed', pay_account)` if it's only user error ?

Reply:
Good Catch..fixed and added a few tests